e2e-tests/latest/endpoints/polkadot.ts

```diff
@@ -141,6 +141,18 @@ export const polkadot: IConfig = {
 		path: '/paras/head/backed-candidates',
 		queryParams: ['at={blockId}'],
 	},
+	'/pallets/{palletId}/dispatchables': {
+		path: '/pallets/democracy/dispatchables',
+		queryParams: [],
+	},
+	'/pallets/{palletId}/dispatchables?onlyIds=true': {
+		path: '/pallets/14/dispatchables',
```

Comment:
Does this test intend to have `onlyIds` as a query param if so it should be added as one. Or is this just to make sure you can test both a integer and name ID?

Reply:
this is mostly to test that the integer id works. we are unable to have duplicate key values here so I added the `onlyIds` as part of the path